en/advanced-gimbal-control.md

```diff
@@ -1,20 +1,24 @@
 # Gimbal Control Setup
 
-PX4 contains a generic mount/gimbal control driver with different input and output methods. Any input can be selected to drive any output.
-
-First, make sure the driver runs, using `vmount start`, then configure its parameters.
+PX4 contains a generic mount/gimbal control driver with different input and
+output methods. Any input method can be selected to drive any output.
 
 ## Parameters
-The parameters are described in [src/drivers/vmount/vmount_params.c](https://github.com/PX4/Firmware/blob/master/src/drivers/vmount/vmount_params.c). The most important ones are the input (`MNT_MODE_IN`) and the output (`MNT_MODE_OUT`) mode. By default, the input is disabled. Any input method can be selected to drive any of the available outputs.
-
-If a mavlink input mode is selected, manual RC input can be enabled in
-addition (`MNT_MAN_CONTROL`). It is active as long as no mavlink message is received yet, or mavlink explicitly requests RC mode.
-
+The parameters are described [here](parameter_reference.md#mount). The most
+important ones are the input (`MNT_MODE_IN`) and the output (`MNT_MODE_OUT`)
```

Comment:
FYI You can actually link specifically to a reference for a particular parameter using its name - e.g. parameter_reference.md#MNT_MODE_IN. The parameter file will probably move shortly, but still worth defining a link and I'll move that too. Up to you.